lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -466,7 +511,7 @@ UsdMayaJobExportArgs::GetDefaultDictionary()
         d[UsdMayaJobExportArgsTokens->eulerFilter] = false;
         d[UsdMayaJobExportArgsTokens->exportCollectionBasedBindings] = false;
         d[UsdMayaJobExportArgsTokens->exportColorSets] = true;
-        d[UsdMayaJobExportArgsTokens->exportDisplayColor] = true;
+        d[UsdMayaJobExportArgsTokens->exportDisplayColor] = false;
```

Comment:
Defaults to false, but can become true when the material export option is "Display Colors" (see mayaUsdTranslatorExport.mel for details).